chainerx_cc/chainerx/routines/evaluation.cc

```diff
@@ -0,0 +1,44 @@
+#include "chainerx/routines/evaluation.h"
+
+#include <cmath>
+#include <cstddef>
+#include <cstdint>
+
+#include <absl/types/optional.h>
+
+#include "chainerx/array.h"
+#include "chainerx/routines/arithmetic.h"
+#include "chainerx/routines/creation.h"
+#include "chainerx/routines/indexing.h"
+#include "chainerx/routines/logic.h"
+#include "chainerx/routines/manipulation.h"
+#include "chainerx/routines/reduction.h"
+#include "chainerx/routines/sorting.h"
+#include "chainerx/routines/statistics.h"
+#include "chainerx/routines/type_util.h"
+#include "chainerx/scalar.h"
+#include "chainerx/shape.h"
+
+namespace chainerx {
+
+Array Accuracy(const Array& x1, const Array& x2, const absl::optional<int8_t>& ignore_label) {
```

Comment:
Let's check the dtype kind of `x1` is a float and `x2` is an integer, and throw an error otherwise.

Reply:
Sure. Modified.